lib/logstash/inputs/jdbc.rb

```diff
@@ -186,6 +187,7 @@ def execute_query(queue)
     # update default parameters
     @parameters['sql_last_start'] = @sql_last_start
     execute_statement(@statement, @parameters) do |row|
+      break if stop?
```

Comment:
would this not be better at the end of the loop? so it checks before actually getting to execute the statement? does this makes sense?